src/CommunityToolkit.Maui.Core/Essentials/SpeechToText/SharedSpeechToTextImplementation.macios.cs

```diff
@@ -21,6 +21,15 @@ public sealed partial class SpeechToTextImplementation
 												? SpeechToTextState.Listening
 												: SpeechToTextState.Stopped;
 
+	/// <summary>
+	/// Initialize <see cref="SpeechToTextImplementation"/>
+	/// </summary>
+	public SpeechToTextImplementation()
+	{
+		var audioSession = AVAudioSession.SharedInstance();
+		audioSession.SetCategory(AVAudioSessionCategory.PlayAndRecord, AVAudioSessionCategoryOptions.DefaultToSpeaker);
```

Comment:
It will be overridden in macOS (see line 32 in mac.cs file).

Reply:
Good catch! What's your recommendation? 

Should we move the logic from the macOS implementation here into the shared constructor of `SpeechToTextImplementation`?

```suggestion
		if (UIDevice.CurrentDevice.CheckSystemVersion(15, 0))
{
  audioSession.SetSupportsMultichannelContent(true, out _);
}

audioSession.SetCategory(AVAudioSessionCategory.Record, AVAudioSessionCategoryOptions.DefaultToSpeaker);
```